fe/fe-core/src/main/java/com/starrocks/common/util/concurrent/MarkedCountDownLatch.java

```diff
@@ -91,6 +91,13 @@ public Status getStatus() {
         return st;
     }
 
+    public synchronized void countDown(int count) {
```

Comment:
why should this interface is added? can the caller predict the number of countdowns and set the initial counts and then wait to `countDownToZero`?

Reply:
deleted